plugin/tooling.py

```diff
@@ -98,7 +98,7 @@ class LspTroubleshootServerCommand(sublime_plugin.WindowCommand, TransportCallba
     def run(self) -> None:
         window = self.window
         active_view = window.active_view()
-        configs = [c for c in windows.lookup(window).get_config_manager().get_configs() if c.enabled]
+        configs = windows.lookup(window).get_config_manager().get_configs()
```

Comment:
This will now show a lot of default-included servers that are disabled.

Ideally we would show only enabled ones and those that are temporarily disabled.

Reply:
I'll revert that.
It's possible to work around the issue it's fixing by restarting ST without triggering servers.